pkg/serverless/protocol.go

```diff
@@ -50,13 +50,21 @@ type Daemon struct {
 	// through configuration.
 	useAdaptiveFlush bool
 
+	// clientLibReady indicates whether the datadog client library has initialised
+	// and called the /hello route on the agent
+	clientLibReady bool
+
+	// datadogLambdaApiUsed tracks whether the layer has called the
+	datadogLambdaApiUsed bool
+
 	// aggregator used by the statsd server
 	aggregator *aggregator.BufferedAggregator
-	stopCh     chan struct{}
 
 	// Wait on this WaitGroup in controllers to be sure that the Daemon is ready.
 	// (i.e. that the DogStatsD server is properly instantiated)
 	ReadyWg *sync.WaitGroup
+
+	InvcWg *sync.WaitGroup
```

Comment:
I would add a docs comment here

Reply:
Also, this is maybe a style thing, but I find code more readable with fewer abbreviations, so I would use `invocationWg`